governance.md

```diff
@@ -1,34 +1,61 @@
+To suggest a change in the ownership of a given repository part, create a pull request with the required changes in the `CODEOWNERS` file in the project's repository. The required number of code owners needs to approve the pull request for the changes to take place. Read [here](https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/template/CODEOWNERS) how to set up and modify owners of the given repository folders and files.
+
+## The decisions-making process
+
+In general, the decisions that affect the project, including its structure, functionalities, components, or work of the project teams are made by the relevant Special Interest Groups (SIGs) and Working Groups (WGs). However, the organizational decisions and those that relate to the product strategy belong to the Kyma Council.
+
+SIGs and WGs follow the "lazy consensus" decision-making process which assumes that:
+
+* All SIG/WG members have an equal voice in the decision-making process.
+* Silence is consent. By default, lack of objections to a proposed decision is interpreted as a silent approval.
+* Any objections are good opportunities for the discussion and healthy, constructive conflicts.
+
+> **NOTE:** The described approach only concerns the decisions taken during SIG and WG meetings. It does not affect any Kyma decisions taken during daily team activities.
+
+### Process
+
+The SIG/WG decison-making process looks as follows:
+
+![Decision-making process](assets/decision-making-process.png)
+
+1. Every decision starts with creating a pull request (PR) with the [**Decision Record**](./guidelines/templates/resources/DR.md) (DR) document that provides the decision details, timelines, status, the related context, and associated consequences. The DR can reference a proposal or any other related document.
+2. The PR creator sends the table with the decision log details to the related SIG/WG distribution list. If the decision input providers are not part of the mailing list, add them to the email communication.
+3. The discussion on the proposed decision can happen through the mailing list, on the related Slack channel, or directly on the pull request. You can also add the topic to the agenda of the upcoming SIG/WG meeting and discuss it with the SIG/WG members. Encourage the discussion and bring up any objections early in the process. A person with a different approach to the topic can create a Counter Proposal.
+4. The discussions on the proposed decision can lead to the change in the actual decision log or the proposal, or end up with no changes required. Update the pull request with the DR with the outcome of the discussion. Otherwise, take actions required to reach a consensus. Those who created a proposal work with those who had the objections to prepare an improved solution or reach a consensus to decline the proposal.
```

Comment:
1. `lead to a change in the actual decision log or`~the~ `proposal` -> isn't there too many "the" ?
2. `Update the DR pull request with the outcome of the discussion.` -> too many "with" made the sentence clumsy
3. `Those who created the proposal`

Reply:
1. One per a noun so I guess that could stay...
2. I left `Update the PR with the outcome of the discussion.`
3. Ok